### PR TITLE
TRUNK-5856: Simplification of the ChangeLogVersionFinder class code

### PR DESCRIPTION
**Pull request by BartlomiejRasztabiga**
## Description of what I changed
TRUNK-5856: Simplify ChangeLogVersionFinder.getSnapshotCombinations method by using ArrayList constructor with parameter

## Issue I worked on
see https://issues.openmrs.org/browse/TRUNK-5856

## Checklist: I completed these to help reviewers :)
- [X] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [X] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [X] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [X] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by sacull** [comment]
@BartlomiejRasztabiga Intro tickets are intended to learn how to use OpenMRS supporting tools such as Github or Jira. Solving a few of them is sufficient for this purpose. Once you have mastered these tools, it's time to tackle the more complicated programming problems, leaving the intro tickets to others. ;)

**Comment by BartlomiejRasztabiga** [comment]
@sacull Of course, trying out more complicated tasks is what I am going to do now :smile:

**Comment by gitcliff** [comment]
@BartlomiejRasztabiga you might wanna look into the travis ci build failure

**Comment by BartlomiejRasztabiga** [comment]
@gitcliff I don't think it's related to this PR as I've run tests locally multiple times to verify this. Can we trigger re-build in Travis?

File: api/src/main/java/org/openmrs/liquibase/ChangeLogVersionFinder.java [hunk 1]
```diff
@@ -88,9 +88,7 @@ public ChangeLogVersionFinder(ChangeLogVersions changeLogVersions) {
 		SortedMap<String, List<String>> changeLogCombinations = new TreeMap<>();
 		
 		for (String snapshotVersion : getSnapshotVersions()) {
-			List<String> changeLogFilenames = new ArrayList<>();
-			
-			changeLogFilenames.addAll(getSnapshotFilenames(snapshotVersion));
+			List<String> changeLogFilenames = new ArrayList<>(getSnapshotFilenames(snapshotVersion));
```
Comment (HerbertYiga) [inline api/src/main/java/org/openmrs/liquibase/ChangeLogVersionFinder.java:91]: does this look good enough to you nowo ? cc @sacull

Comment (sacull) [reply api/src/main/java/org/openmrs/liquibase/ChangeLogVersionFinder.java:91]: @HerbertYiga I don't think you need to wait for my answer. How do you think about it yourself? Is this PR ready to merge or not yet? ;)

**Comment by gracepotma** [comment]
@sacull can this one be approved?

**Comment by sherrif10** [comment]
Hello @BartlomiejRasztabiga  any progress on this

**Comment by BartlomiejRasztabiga** [comment]
@sherrif10 It's waiting to be reviewed by the ticket reporterr, @sacull, since March 15 :)

**Comment by sherrif10** [comment]
cc @dkayiwa  @ibacher  , Sorry i think @sacull  might be little bussy at the moment but am sure he will come back

**Pull request merged**